Readme.md

```diff
@@ -212,6 +212,15 @@ gh.deployments("foo/bar").status deployId, (deploys) ->
   console.log deploys[0].state
 ```
 
+### Pull Requests ###
+
+#### Merge a Pull Request ####
+
+```coffeescript
+gh.pullrequest("foo/bar").pr_merge prNumber, (result) ->
+  console.log result.sha
```

Comment:
Just a couple stylistic changes, can we change `pullrequest` to `pull_requests`, and `pr_merge` to `merge`? So this call would become:

 ```coffeescript
gh.pull_requests("foo/bar").merge prNumber, (result) ->
  console.log result.sha
```